csuntechlab/iSTART
# HOTFIX-03-05-2019--fixResponsiveBreakpoints

# Description
Fixed the responsive breakpoints using the bootstrap system; Fixed linter errors; Fixed margins for login; Changed component names to conform with our convention.

# Testing
1. Visit the login page.
2. Ensure when viewing on a smaller device(iPhone 5/SE) that the login has proper margins.
